routes/recipesRouter.js

```diff
@@ -3,3 +3,9 @@ import express from "express";
 import { recipesControllers } from "../controllers/recipesControllers.js";
 
 export const recipesRouter = express.Router();
+
+recipesRouter.get("/", recipesControllers.getFilteredRecipesController);
```

Comment:
Add a middleware `validateQueryString` and create a schema to validate incoming filters in the query string.